client/src/containers/NewsOfTheDays/component/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Loader from 'react-loader-spinner';
+import './style.scss';
+import { fetchData } from '../action';
+import { connect } from 'react-redux';
+
+
+class NewsOfTheDays extends Component {
+  //Create pass/link to news page to use this function
+  /*const showSeeMoreButton = (text, fullNews) => (
+    text.length < 100 ? text
+      : <p>{text.slice(0, 60)}<Link to={fullNews} className='see-more'> ...</Link></p>
+  );*/
+
+  componentDidMount() {
+    const { dispatch } = this.props;
+
+    dispatch(fetchData());
+  }
+
+  showSeeMoreButton(text) {
+    return text.length < 100 ? text
+      : <p>{text.slice(0, 60)}<Link to='/' className='see-more'> ...</Link></p>;
+  };
+
+  render() {
+    const { data } = this.props;
+
+    return (
+      data.length === 0
```

Comment:
As far as I know return requires one wrapper. How does it work here?

Reply:
! Data.Length

Reply:
answered in telegram